x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/ml_inference/model_select_logic.ts

```diff
@@ -126,18 +125,20 @@ export const ModelSelectLogic = kea<MakeLogicType<ModelSelectValues, ModelSelect
     createModel: ({ modelId }) => {
       actions.createModelMakeRequest({ modelId });
     },
-    createModelSuccess: () => {
+    createModelSuccess: (response) => {
       actions.startPollingModels();
+      // The create action succeeded, so the model is no longer a placeholder
+      actions.setSelectedModelNotPlaceholderFromMLInferenceLogic(response.modelId);
```

Comment:
Can `response` ever be empty/no model ID? (I assume no because this is successful but double checking)

Reply:
It cannot be nullish according to the [CreateModelResponse](https://github.com/elastic/kibana/blob/fea95f0725dd28a2d482959ff367f999ad1443ca/x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/api/ml_models/create_model_api_logic.ts#L16) type.